README.adoc

```diff
@@ -54,6 +54,11 @@ kubectl apply -f config
 ```
 kubectl create -f samples/greetings-topic.yaml
 ```
+=== Build the Docker container for the function code
+
+```
+./faas create -f scripts -v v0001 -p samples/script/
```

Comment:
this should be `-f echo`

Reply:
but then needs to create a second image for wordcount

Reply:
yes, that's exactly what we want for the pre-baked option ... we will revisit the dynamic option (where the base image is sufficient)

Reply:
I will change while merging.